src/resources/reducer.js

```diff
@@ -0,0 +1,18 @@
+import { combineReducers } from 'redux';
+
+import { USER_LOGOUT } from './user/user.actions';
+import user from './user/user.reducer';
+
+const appReducer = combineReducers({
+  user,
+});
+
+const rootReducer = (state, action) => {
+  if (action.type === USER_LOGOUT) {
+    // eslint-disable-next-line no-param-reassign
+    state = undefined;
```

Comment:
@SobakaSlava can we replace `state = undefined` with `state = null` unless there is some reason behind undefined? Assigning to `undefined` feels unnatural

Reply:
No, it's not possible to assign `state` to `null`. The reason behind `undefined` is that the inner logic of redux assumes that assigning `state = undefined` leads to initializing the state with initial values. That's the approach by Dan Abramov: https://stackoverflow.com/questions/35622588/how-to-reset-the-state-of-a-redux-store
I've checked and it doesn't work with `null`